sdk/eventhub/Directory.Build.props

```diff
@@ -11,6 +11,21 @@
     <IsSamplesProject>true</IsSamplesProject>
     <IsTestSupportProject>true</IsTestSupportProject>
   </PropertyGroup>
+  
+  <!-- 
+    OptOut of the global NUnit test timeout for all Event Hubs libraries.
+    
+    The approach that NUnit uses to measure timing for tests does not play well with parallel execution
+    and highly asynchronous code.  Many of the Event Hubs tests run against non-deterministic background
+    operations where timing is longer than normal unit tests and the exact timing is difficult to pin down.
+    
+    Tests currently use an environment variable with cancellation-based timeout control, which allows for
+    more generous timeouts in the CI environments, where resources are often more constrained and 
+    continuations take longer to be scheduled.    
+  -->
+  <PropertyGroup>
+        <IsOptOutOfGlobalTestTimeout>true</IsOptOutOfGlobalTestTimeout>
```

Comment:
Should we allow increasing timeouts instead of complete opt-out?

Reply:
I think OptOut should be considered test debt that needs to be fixed at some point. I only used this in cases where the work to address the timeout issues was too much for this PR. Eventhubs is the best example of this, because of how different its use of test infrastructure is.